src/NRules/NRules.Testing/CachedRuleActivator.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NRules.Fluent;
+using NRules.Fluent.Dsl;
+
+namespace NRules.Testing;
```

Comment:
@gritcsenko I'm still internally debating between NRules.Test and NRules.Testing. I'll think a bit more. Do you think NRules.Test will cause name clashes?

Reply:
@snikolayev I think namespaces should be named as goups or collections, not as single entities to avoid name clashes with class names.